SIL.WritingSystems.Tests/SldrTests.cs

```diff
@@ -264,17 +272,52 @@ public void GetLdmlFile_NotModified_DoesntDownloadNewFile()
 			File.WriteAllText(Path.Combine(environment.FilePath, ietfLanguageTag + ".ldml"), content);
 			File.WriteAllText(Path.Combine(Sldr.SldrCachePath, ietfLanguageTag + ".ldml"), content);
 
-			var sldrStatus = environment.GetLdmlFile(ietfLanguageTag, out var filename);
-			if (sldrStatus == SldrStatus.UnableToConnectToSldr)
-				Assert.Ignore("Ignored because SLDR is offline.");
+			environment.TryGetLdmlFile(ietfLanguageTag, out var filename);
 			// Call a second time, this should use the Cache now. However, this still reports it as FromSldr
-			sldrStatus = environment.GetLdmlFile(ietfLanguageTag, out filename);
+			var sldrStatus = environment.GetLdmlFile(ietfLanguageTag, out filename);
 			Assert.That(sldrStatus, Is.EqualTo(SldrStatus.FromSldr));
 			AssertThatXmlIn.File(Path.Combine(environment.FilePath, filename))
 				.HasAtLeastOneMatchForXpath("/ldml/identity/special/sil:identity[@source='cldr']", environment.NamespaceManager);
 		}
 
-		#region internal methods
+		[Test]
+		[Category("LongRunning")] // ~2 minutes
+		public void GetLdmlFile_GetsValidLDML([Values("false", "true")] string isStaging,
+			[Values("ar", "az", "bn", "de", "en", "es", "en-GB", "fa", "fr", "hi", "hu", "id", "km", "ko", "ml", "ms", "my",
+				"ne", "pt", "ru", "rw", "sw", "ta", "te", "th", "tr", "ur", "vi", "zh", "zh-CN")] string ietfLanguageTag)
+		{
+			DownloadAndVerifyLDML(isStaging, ietfLanguageTag);
+		}
+
+		[Test]
```

Comment:
Could add a comment that this test runs with every build whereas we don't normally run the above test in CI builds.